hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/BaseSparkCommitActionExecutor.java

```diff
@@ -89,6 +91,13 @@
   private static final Logger LOG = LoggerFactory.getLogger(BaseSparkCommitActionExecutor.class);
   protected final Option<BaseKeyGenerator> keyGeneratorOpt;
 
+  static {
+    // JsonUtils for Support Spark Version >= 3.3
+    if (HoodieSparkUtils.gteqSpark3_3()) {
+      JsonUtils.registerModules();
+    }
+  }
```

Comment:
Does Spark read side need this?  This class only handles write side, correct?

Reply:
from error stack，report write side is conflict @yihua

Reply:
> This class only handles write side, correct?

i would put the static code to spark common class @yihua